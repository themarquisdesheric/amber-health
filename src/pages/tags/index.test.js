import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Tags from './';

const data = {
  site: {
    siteMetadata: {
      title: 'The Chronic'
    }
  },
  allMarkdownRemark: {
    group: [
      {
        fieldValue: 'EDS',
        totalCount: 1
      },
      {
        fieldValue: 'Ehlers-Danlos Syndrome',
        totalCount: 1
      },
      {
        fieldValue: 'IC',
        totalCount: 1
      },
      {
        fieldValue: 'Interstitial cystitis',
        totalCount: 1
      },
      {
        fieldValue: 'PFD',
        totalCount: 1
      },
      {
        fieldValue: 'Urogenital',
        totalCount: 2
      },
      {
        fieldValue: 'autoimmune',
        totalCount: 1
      },
      {
        fieldValue: 'bladder',
        totalCount: 1
      },
      {
        fieldValue: 'book',
        totalCount: 1
      },
      {
        fieldValue: 'book review',
        totalCount: 1
      },
      {
        fieldValue: 'chronic illness',
        totalCount: 1
      },
      {
        fieldValue: 'chronic pain',
        totalCount: 2
      },
      {
        fieldValue: 'disease',
        totalCount: 1
      },
      {
        fieldValue: 'endo',
        totalCount: 1
      },
      {
        fieldValue: 'endometriosis',
        totalCount: 6
      },
      {
        fieldValue: 'genital',
        totalCount: 1
      },
      {
        fieldValue: 'hypermobile',
        totalCount: 1
      },
      {
        fieldValue: 'lichen sclerosus',
        totalCount: 1
      },
      {
        fieldValue: 'pain',
        totalCount: 1
      },
      {
        fieldValue: 'painful bladder syndrome',
        totalCount: 1
      },
      {
        fieldValue: 'pelvic floor dysfunction',
        totalCount: 1
      },
      {
        fieldValue: 'skin',
        totalCount: 1
      },
      {
        fieldValue: 'uterus',
        totalCount: 1
      },
      {
        fieldValue: 'women\'s health',
        totalCount: 7
      }
    ]
  }
};

test('Tags', () => {
  const wrapper = shallow(<Tags data={data} />);

  expect(toJson(wrapper)).toMatchSnapshot();
});
