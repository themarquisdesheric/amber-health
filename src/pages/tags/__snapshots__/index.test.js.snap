// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tags 1`] = `
<TemplateWrapper>
  <section
    className="content"
  >
    <Breadcrumbs
      path="Tags"
    />
    <div
      className="flex flex-col items-center"
    >
      <div
        className="max-w-xl"
      >
        <HelmetWrapper
          defer={true}
          encodeSpecialCharacters={true}
          title="Tags | The Chronic"
        />
        <div>
          <div
            className="mb-24"
          >
            <ul>
              <li
                className="tag"
                key="EDS"
              >
                <mockConstructor
                  to="/tags/eds/"
                >
                  EDS
                   (
                  1
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="Ehlers-Danlos Syndrome"
              >
                <mockConstructor
                  to="/tags/ehlers-danlos-syndrome/"
                >
                  Ehlers-Danlos Syndrome
                   (
                  1
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="IC"
              >
                <mockConstructor
                  to="/tags/ic/"
                >
                  IC
                   (
                  1
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="Interstitial cystitis"
              >
                <mockConstructor
                  to="/tags/interstitial-cystitis/"
                >
                  Interstitial cystitis
                   (
                  1
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="PFD"
              >
                <mockConstructor
                  to="/tags/pfd/"
                >
                  PFD
                   (
                  1
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="Urogenital"
              >
                <mockConstructor
                  to="/tags/urogenital/"
                >
                  Urogenital
                   (
                  2
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="autoimmune"
              >
                <mockConstructor
                  to="/tags/autoimmune/"
                >
                  autoimmune
                   (
                  1
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="bladder"
              >
                <mockConstructor
                  to="/tags/bladder/"
                >
                  bladder
                   (
                  1
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="book"
              >
                <mockConstructor
                  to="/tags/book/"
                >
                  book
                   (
                  1
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="book review"
              >
                <mockConstructor
                  to="/tags/book-review/"
                >
                  book review
                   (
                  1
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="chronic illness"
              >
                <mockConstructor
                  to="/tags/chronic-illness/"
                >
                  chronic illness
                   (
                  1
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="chronic pain"
              >
                <mockConstructor
                  to="/tags/chronic-pain/"
                >
                  chronic pain
                   (
                  2
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="disease"
              >
                <mockConstructor
                  to="/tags/disease/"
                >
                  disease
                   (
                  1
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="endo"
              >
                <mockConstructor
                  to="/tags/endo/"
                >
                  endo
                   (
                  1
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="endometriosis"
              >
                <mockConstructor
                  to="/tags/endometriosis/"
                >
                  endometriosis
                   (
                  6
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="genital"
              >
                <mockConstructor
                  to="/tags/genital/"
                >
                  genital
                   (
                  1
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="hypermobile"
              >
                <mockConstructor
                  to="/tags/hypermobile/"
                >
                  hypermobile
                   (
                  1
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="lichen sclerosus"
              >
                <mockConstructor
                  to="/tags/lichen-sclerosus/"
                >
                  lichen sclerosus
                   (
                  1
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="pain"
              >
                <mockConstructor
                  to="/tags/pain/"
                >
                  pain
                   (
                  1
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="painful bladder syndrome"
              >
                <mockConstructor
                  to="/tags/painful-bladder-syndrome/"
                >
                  painful bladder syndrome
                   (
                  1
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="pelvic floor dysfunction"
              >
                <mockConstructor
                  to="/tags/pelvic-floor-dysfunction/"
                >
                  pelvic floor dysfunction
                   (
                  1
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="skin"
              >
                <mockConstructor
                  to="/tags/skin/"
                >
                  skin
                   (
                  1
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="uterus"
              >
                <mockConstructor
                  to="/tags/uterus/"
                >
                  uterus
                   (
                  1
                  )
                </mockConstructor>
              </li>
              <li
                className="tag"
                key="women's health"
              >
                <mockConstructor
                  to="/tags/womens-health/"
                >
                  women's health
                   (
                  7
                  )
                </mockConstructor>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
  </section>
</TemplateWrapper>
`;
